jstz_node/src/services/accounts.rs

```diff
@@ -29,11 +39,60 @@ async fn nonce(
     Ok(HttpResponse::Ok().json(nonce))
 }
 
+#[get("/{address}/kv")]
+async fn kv(
+    rollup_client: Data<RollupClient>,
+    path: Path<String>,
+    query: web::Query<HashMap<String, String>>,
```

Comment:
Nit: Here I think it is clearer (and more consistent) to use a top-level import (avoiding the qualified `web`)

```suggestion
    query: Query<HashMap<String, String>>,
```